tests/Escaper/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+project(EscaperTests VERSION 1.0.0 LANGUAGES CXX)
+
+add_library(EscaperTests SHARED)
+target_sources(EscaperTests
+    PRIVATE     EscaperTests.cpp)
+target_link_libraries(EscaperTests
+    PRIVATE     TASTE::Escaper
+    PUBLIC      CppUTest
+                pthread)
```

Comment:
pthread shall not be required here (btw, it shall be Threads::Threads in platform independent project).

Reply:
Ok, it is removed now.